source/backend/api/Areas/Tools/Controllers/LtsaController.cs

```diff
@@ -130,6 +141,59 @@ public async Task<IActionResult> PostLtsaFields(string pid)
             var result = await _ltsaService.PostLtsaFields(pid);
             return new JsonResult(result);
         }
+
+        /// <summary>
+        /// Upload a newline delimited list of pid numbers reflecting BCTFA ownership as per LTSA.
+        /// </summary>
+        /// <param name="ownershipFile">the file containing the list of pids.</param>
+        /// <returns>The orders created within LTSA.</returns>
+        [HttpPut("ownership")]
+        [Produces("application/json")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(typeof(Pims.Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "tools-ltsa" })]
+        [HasPermission(Permissions.LtsaOwnershipEdit)]
+        public async Task<IActionResult> PutBctfaOwnership(IFormFile ownershipFile)
```

Comment:
did they say anything about being able to do a call with JSON instead of a file?

Reply:
Not yet, this is the first time they would offer what they call a "data product" as a direct rest api call. In the past they have only supported SFTP uploads.